Update FAQ with definition of ancient DNA (#39)
MIxS-MInAS/MIxS-MInAS.github.io: docs/documentation/user-documentation/faq.md

## What do we define as ancient DNA?

> There is no single answer to this question. Ancient DNA is not strictly defined by age but rather by its condition - characterised by post-mortem molecular damage.

Reviewer: ```suggestion
There is no single answer to this question. 
Ancient DNA is not strictly defined by age but rather by its condition - characterised by post-mortem molecular damage.
```

In general, ancient DNA is any DNA derived from samples that:
  - had not been prepared specifically for DNA analyses,
  - were only considered for molecular analysis long after the organism died,
  - require special handling due to expected specific forms of degradation,

>   - for which the age of the sample is not a nuisance parameter.

Reviewer: It's not clear to me what is meant by the last point. Can you try to expand or rephrase a bit?

> This definition will not, however, apply in all cases. We are not able to _a priori_ know the level of molecular damage for any given material, whereas qualification as "ancient DNA sample" should be applicable to a sample even before it gets analysed. There will also be some ancient DNA samples that show little to no degradation as well as ancient DNA that do not strictly apply to any particular "organism" that had died at some particular time (e.g., sedaDNA). Additionally, for some datasets only some samples will qualify as "ancient" but still it will be practical to apply "ancient" metadata checklist across the whole dataset to ensure consistency and coherence.

Reviewer: ```suggestion
This definition will not, however, apply in all cases. 
We are not able to _a priori_ know the level of molecular damage for any given material, whereas qualification as "ancient DNA sample" should be applicable to a sample even before it gets analysed. 
There will also be some ancient DNA samples that show little to no degradation as well as ancient DNA that does not strictly apply to any particular "organism" that had died at some particular time (e.g., sedaDNA). 
Additionally, for some datasets only some samples will qualify as "ancient" but still it will be practical to apply "ancient" metadata checklist across the whole dataset to ensure consistency and coherence.
```

## What do we define as ancient DNA?

There is no single answer to this question. 
Ancient DNA is not strictly defined by age but rather by its condition - characterised by post-mortem molecular damage.

> In general, ancient DNA is any DNA derived from samples that:

Reviewer: ```suggestion
In general, ancient DNA is any DNA derived from samples:
```

>   - had not been prepared specifically for DNA analyses,

Reviewer: ```suggestion
  - that had not been prepared specifically for DNA analyses,
```

>   - were only considered for molecular analysis long after the organism died,

Reviewer: ```suggestion
  - that were only considered for molecular analysis long after the organism died,
```

>   - require special handling due to expected specific forms of degradation,

Reviewer: ```suggestion
  - that require special handling due to expected specific forms of degradation,
```

>   - for which the age of the sample is not a nuisance parameter.

Reviewer: ```suggestion
  - for which the age of the sample is a crucial piece of information which affects its processing and analysis directly.
```
